app/en/home/compare-server-types/page.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Compare Server Types"
+description: "Compare the different types of MCP servers"
+---
+
+Depending on the transport you use and where you want to run your MCP server, Arcade offers different functionalities and features. Below is a comparison of the different server types and their capabilities.
+
+Server types:
+
+| Transport | Deployment                      | Description                                                                                                                                                                                               |
+| --------- | ------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| stdio     | local                           | MCP Server running as a subprocess of the client. Local only.                                                                                                                                             |
+| http      | local (unprotected)             | MCP server running on a local network, accessible from the local machine, without server-level auth.                                                                                                      |
```

Comment:
we could add these terms (protected vs unprotected) to our glossary so that they can hover on it on this page and see what they mean. Could be useful to link out to the official MCP specification/docs

Reply:
Based on the slack conversation, I moved the definitions to the glossary, and removed this table